src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = readData(fromFileName);
+        writeToFile(report, toFileName);
```

Comment:
You miss report creating operation.
![image](https://user-images.githubusercontent.com/20326858/220094946-58294253-a7bf-4bcb-8cd6-46975a57bee9.png)